logging/logging.go

```diff
@@ -134,11 +134,19 @@ type Client struct {
 // By default NewClient uses WriteScope. To use a different scope, call
 // NewClient using a WithScopes option (see https://godoc.org/google.golang.org/api/option#WithScopes).
 func NewClient(ctx context.Context, parent string, opts ...option.ClientOption) (*Client, error) {
-	parent = makeParent(parent)
+	var (
+		c *vkit.Client
```

Comment:
declare these just in time with `:=`

Reply:
The problem there is with `err` which has to be declared twice if I use `:=`